src/cascadia/TerminalApp/TerminalSettingsSerializationHelpers.h

```diff
@@ -180,13 +180,11 @@ JSON_ENUM_MAPPER(::winrt::Microsoft::UI::Xaml::Controls::TabViewWidthMode)
     };
 };
 
-DEFINE_ENUM_FLAG_OPERATORS(::winrt::Microsoft::Terminal::Settings::CopyFormat);
-
-JSON_FLAG_MAPPER(::winrt::Microsoft::Terminal::Settings::CopyFormat)
+JSON_FLAG_MAPPER(::winrt::Microsoft::Terminal::TerminalControl::CopyFormat)
 {
     JSON_MAPPINGS(5) = {
         pair_type{ "none", AllClear },
-        pair_type{ "plain", ValueType::Plain },
+        pair_type{ "plain", AllClear },

```

Comment:
because of how flag mapper works, this will work badly if you say `formats: [plain, html]` (it'll throw an error)

Perhaps we can just leave plain off all the time? Just plain isn't even a _format_?